hooks/useNotifications.tsx

```diff
@@ -13,25 +13,28 @@ import { useRef } from "react";
 import { useProvider } from "wagmi";
 
 const toastDefaultArgs = {
-  position: "top-right" as ToastPosition,
+  position: "bottom-right" as ToastPosition,
   duration: 5000,
   isClosable: true,
 };
 
+const ToastLink = ({ href, ...props }: ToastProps & { href: string }) => (
+  <Link href={href} isExternal>
```

Comment:
because the Link wraps the whole toast, a "dismiss" click takes us to the etherscan link.

Reply:
Thanks! Wrapped the description with the link instead and remove the whole custom toast link component

Reply:
Also, removed closability of the pending toast  as that prevents the success toast from showing